Surface_mesh_shortest_path/doc/Surface_mesh_shortest_path/Concepts/SurfaceMeshShortestPathTraits.h

```diff
@@ -247,31 +247,31 @@ class SurfaceMeshShortestPathTraits
 
   /*!
   Function object type that provides
-  `Barycentric_coordinate operator()(FT a, FT b, FT c)`
-  to introduce a new triangular barycentric coordinate.
+  `Barycentric_coordinates operator()(FT a, FT b, FT c)`
+  to introduce some new triangular barycentric coordinates.
```

Comment:
Random note: The use of the term `some` feels a little weird to me. Its technically correct, but it feels oddly informal. You might consider `to construct new triangular barycentric coordinates` instead, but its just my gut reaction, I don't have a specific reason to prefer it (and its definitely not a big deal).